### ISSUE TITLE
Please update all pouch, redux, and react dependencies to latest versions

### ISSUE THREAD

**Issue by mandysimon88** [pgte/pouch-websocket-sync-example#4]
Just spent about 32 man hours using this example repo to try to create a similar app... We ran into several version compatibility problems and had to roll this back and forward, with bugs appearing and disappearing depending on versions used. Between the most recent and where your repo is locked, we were not able to find a magic combination where "everything works, out of the box".

(I created a separate issue for the one that causes a fresh install of this repo to break.)

I know you haven't updated this repo in 11 months... but it would be really nice if it had the latest versions of these key dependencies.

Thank you!

**Comment by rxrossi** [comment]
Hi @mandysimon88, I would like to know if you managed to make it work, I'm getting some errors that I have no clue how to fix, do you have some guidance?

**Comment by mandysimon88** [comment]
No, unfortunately, we were not able to make it work. We did an 8-hour spike with it and then didn’t touch Pouch after that. We experimented with walking dependencies forward and back but never found a winning combination for eliminating all errors.

If this example still doesn’t run on download/install, my suggestion is to look at another stack because this one isn’t being maintained fast enough for production use.

**Comment by dcryan** [comment]
I can get the example working with node v4.7.0, but  I get the following error when I use any higher version. Is this the same issue you guys are coming across? Webpack seems to be blowing up on something that I can't follow.

```
inherits_browser.js:5 Uncaught TypeError: Object prototype may only be an Object or null: undefined
    at Function.create (<anonymous>)
    at inherits (inherits_browser.js:5)
    at Object.<anonymous> (stream.js:45)
    at __webpack_require__ (bootstrap f29e410…:555)
    at fn (bootstrap f29e410…:86)
    at Object.<anonymous> (server.js:3)
    at __webpack_require__ (bootstrap f29e410…:555)
    at fn (bootstrap f29e410…:86)
    at Object.<anonymous> (index.js:3)
    at __webpack_require__ (bootstrap f29e410…:555)
```
